instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_server.py

```diff
@@ -113,8 +113,9 @@ def set_trailing_metadata(self, *args, **kwargs):
     def abort(self, code, details):
         self.code = code
         self.details = details
+        self._active_span.set_attribute("rpc.status_code", code.name)
         self._active_span.set_status(
-            Status(status_code=StatusCode(code.value[0]), description=details)
```

Comment:
It wasn't caught because there aren't any real tests for error conditions in this instrumentation.

I wouldn't be opposed to working on that part also, but being thorough is going to be a bunch of work, and it seemed prudent to add a quick fix since the previous PR now violates the recently-changed spec.